app/utils/sweats_message_handler.js

```diff
@@ -0,0 +1,141 @@
+import { DiscordRequest } from './discord_request.js';
+import { EventEmitter } from 'events';
+const green=0x66FF33
+const red=0xff6633
+
+export async function sweatsMessageHandler() {
+    try {
+        const channelId = process.env.CHANNEL_ID_CSWEATS;
+        const messageId = await sendMessage(channelId);
+        const intervalId = listenForReactions(channelId, messageId);
+        if (intervalId === null) {
+            return;
+        }
+        setTimeout(() => clearInterval(intervalId), 1000 * 60 * 60 * 24); // 24 hours
+    } catch (error) {
+        console.error(error);
+        throw new Error('Error handling sweats message');
+    }
+}
+
+
+function listenForReactions(channelId, messageId) {
+    const endpoint = `channels/${channelId}/messages/${messageId}/reactions/%F0%9F%92%A7`;
```

Comment:
(Suggestion: architecture) Is `/%F0%9F%92%A7` specifically for the sweatdrop emoji or a colour? If so, I think we can add this as a parameter to this function and use it there instead of hard coding it